.storybook/storybook.scss

```diff
@@ -11,3 +11,17 @@ table {
     border: 1px solid;
   }
 }
+
+.button-container {
+  border: 1px dotted black;
+  margin-right: 20px;
+  width: 122px;
+  float: left;
```

Comment:
How about we try to not use floats and just use flex or grid?

Reply:
In general: yes.  For this: unless there's another way to do it that I'm not thinking of, it would necessitate adding a whole new element with its own classname to be a parent for these button-container elements (so that we would have something to set `display: flex` or `display: grid` on).  Sure it's worth it?

Reply:
Yeah, I don't think having an extra container for better css is a bad thing.